packages/core/src/caches/well-known.ts

```diff
@@ -52,6 +53,9 @@ function getValueGuard(type: WellKnownCacheType): ZodType<WellKnownMap[typeof ty
     case 'custom-phrases': {
       return z.record(z.unknown());
     }
+    case 'tenant-cache-expires-at': {
+      return z.number();
```

Comment:
can use an object instead of a number?

Reply:
Why using an object?

Reply:
it seems the response type in an object can easily be extended, but it's fine to keep as it is since it's for internal use.
e.g. can make the type to be 'tenant-cache' and the response to be `z.object({ expiresAt: z.number().optional(), createdAt: z.number() })`. In this case, you do not need to read `createdAt` from the tenant instance.

Reply:
No, the `createdAt` must be associated with the tenant instance, since different cloud cluster instance will have different cached tenant instance, hence the createdAt also varies from instance to instance. If you put the `createdAt` in redis cache, you don't know which tenant instance it belongs to.